yunwoo/programmers-1/수박수박수박수박수박수.js

```diff
@@ -0,0 +1,31 @@
+function solution(n) {
+  return n % 2 === 0 ? "수박".repeat(n / 2) : "수박".repeat(n / 2) + "수";
+}
```

Comment:
```suggestion
function solution(n) {
  const repeatedWatermelon = "수박".repeat(n / 2);
  return n % 2 === 0 ? repeatedWatermelon : repeatedWatermelon + "수";
}
```
- 잘푸셨습니다 👍 피드백을 드리자면 `"수박".repeat(n / 2)`와 같이 반복되는 값은 변수로 빼주시는게 좋습니다

Reply:
넵 참고해보겠습니다 리뷰 감사합니다!!👍👍